src/lib/arch/riscv/InstructionMetadata.cc

```diff
@@ -549,9 +554,6 @@ void InstructionMetadata::convertCompressedInstruction(const cs_insn& insn) {
       // C.LI rd, imm, _ -> addi rd, zero, imm
 
       // rd = zero encodes hints
```

Comment:
I assume this possible decoding is no longer possible?

Reply:
Rather than write this comment multiple times, there are several cases in this file where asserts are removed but don't seem to be replaced with an `illegalAlias` call. Just double checking they're all intended

Reply:
This is intentional. While these instructions encode hints which we do not support, the instructions themselves aren't illegal and should pass through the pipeline. Looking closely, all instructions encoding hints have Rd = X0 i.e. the destination register is the zero register. The result of any operation that these instructions perform is discarded so they can safely flow through the pipeline and have no architectural effect